crates/sui-indexer/src/types.rs

```diff
@@ -356,12 +356,13 @@ pub struct IndexedTransaction {
 #[derive(Debug, Clone)]
 pub struct TxIndex {
     pub tx_sequence_number: u64,
+    pub tx_kind: TransactionKind,
     pub transaction_digest: TransactionDigest,
     pub checkpoint_sequence_number: u64,
     pub input_objects: Vec<ObjectID>,
     pub changed_objects: Vec<ObjectID>,
     pub payers: Vec<SuiAddress>,
-    pub senders: Vec<SuiAddress>,
+    pub sender: SuiAddress,
```

Comment:
I vaguely recall that we did `SuiAddress` at the beginning and changed it to Vec later b/c tx was going to have multiple senders. @amnn do you know this either way?

Reply:
Yes, that's right -- this was in anticipation of some potential work for supporting transactions signed by multiple keys (a generalisation of sponsorship), but that hasn't materialised yet, and given the complexities related to indexing in the presence of multiple senders, I thought it was best to revert back to the model that JSON-RPC uses in this case and revisit multiple senders when we need to.